.golangci.yml

```diff
@@ -48,7 +48,6 @@ output:
 linters:
   enable:
     - bodyclose
-    - deadcode
```

Comment:
deadcode doesn't sound related to field alignment, which makes me curious as to why this line was removed.

Reply:
`deadcode` is replaced by `unused` which is enabled by default. https://golangci-lint.run/usage/linters/

Reply:
^ It's in the "along the way" of the commit but I didn't copy that language to the PR after I update it.

Reply:
👍